plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerColumnHandle.java

```diff
@@ -15,21 +15,49 @@
 package io.trino.plugin.faker;
 
 import io.trino.spi.connector.ColumnHandle;
+import io.trino.spi.predicate.Range;
 import io.trino.spi.type.Type;
 
+import java.util.List;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
 import static java.util.Objects.requireNonNull;
 
 public record FakerColumnHandle(
         int columnIndex,
         String name,
         Type type,
         double nullProbability,
-        String generator)
+        String generator,
+        String min,
+        String max,
+        List<String> options)
```

Comment:
Unless you want to directly use these somewhere, it would be better for range and parsedValues to be the member variables

Reply:
`FakerColumnHandle` needs to be serializable, if I use `Range` as a member, I get:
```
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class io.trino.spi.predicate.Range
```

The same applies to options (values), not all types are serializable, like `io.trino.spi.type.LongTimestampWithTimeZone`.

Reply:
Can you try with `io.trino.spi.predicate.Domain#create`, I'm sure we serialize domains for transfer between coordinator and worker

Reply:
It worked! I'm not very comfortable working with domains, and I wouldn't have figured this out myself, thanks!